tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -267,18 +266,19 @@ def test_slurm_pmix(pcluster_config_reader, scheduler, clusters_factory, use_log
     remote_command_executor = RemoteCommandExecutor(cluster, use_login_node=use_login_node)
 
     # Ensure the expected PMIx version is listed when running `srun --mpi=list`.
-    # Since we're installing PMIx v3.1.5, we expect to see pmix and pmix_v3 in the output.
     # Sample output:
     # [ec2-user@ip-172-31-33-187 ~]$ srun 2>&1 --mpi=list
     # srun: MPI types are...
     # srun: none
     # srun: openmpi
     # srun: pmi2
     # srun: pmix
-    # srun: pmix_v3
+    # srun: pmix_vX
+    #
+    # _vX is the Major number of the PMIx version installed and used to compile slurm.
+    # We check this in the cookbook, so we do not repeat the check here
     mpi_list_output = remote_command_executor.run_remote_command("srun 2>&1 --mpi=list").stdout
     assert_that(mpi_list_output).matches(r"\s+pmix($|\s+)")
```

Comment:
I would say that these are also canary-like tests and we don't have to test them here. But that's probably something for the future.